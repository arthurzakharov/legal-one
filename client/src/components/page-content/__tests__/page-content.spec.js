import React from 'react';
import renderer from 'react-test-renderer';
import PageContent from '../page-content.component';

describe('PageContent: ', () => {
  test('render', () => {
    const component = renderer
      .create(
        <PageContent>
          <p>x</p>
        </PageContent>
      )
      .toJSON();
    expect(component).toMatchSnapshot();
  });
});
