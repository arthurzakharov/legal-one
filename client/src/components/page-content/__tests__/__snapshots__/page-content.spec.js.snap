// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`PageContent:  render 1`] = `
<div
  className="PageContent"
>
  <p>
    x
  </p>
</div>
`;
